File: SpringMicroservices/microservice-notes/src/main/java/com/microservice/notes/infrastructure/adapters/in/rest/dto/NoteDto.java
package com.microservice.notes.infrastructure.adapters.in.rest.dto;

import lombok.*;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class NoteDto {
    private Long id;
    private String title;
    private String content;
    private String userId;
    private Long notebookId;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: ProyFronted/app/src/main/java/com/example/proyfronted/backend/Notes/dto/NoteDto.kt
package com.example.proyfronted.backend.Notes.dto

data class NoteDto(
    val id: Long? = null,
    val title: String,
    val content: String,
    val notebookId: Long,
    val createdAt: String? = null,
    val updatedAt: String? = null
)
